docs/api/cli-options.md

```diff
@@ -1,14 +1,26 @@
-# `roca` CLI Options
+# `roca` CLI
+
+## Reporting
 
 Roca exclusively reports its state via the [Test Anything Protocol](http://testanything.org/), and defaults to a Mocha-like "spec" output.  Failed tests cause the `roca` CLI to return a non-zero exit code, which allows most continuous integration systems to automatically detect pass/fail states.
 
 Other output formats are available!  See `roca --help` for more details.
 
+## Usage
+
+```shell
+roca [options] [file names and/or patterns]
+```
+
+For more information, see [Running from the command line](getting-started/quick-start#running-from-the-command-line).
+
+## Options
+
 | Option                  | Description       |
 | ------------------------|----------------|
 | `-h`/`--help`           | The help menu. |
 | `-s`/`--source`         | Path to brs files (if different from `source/`) |
-| `-R`/`--reporter`       | The mocha reporter to use. See [the `mocha` docs](https://mochajs.org/#reporters) for a full list of reporters and example output. _Note: we use the [`tap-mocha-reporter`](https://github.com/tapjs/tap-mocha-reporter) NPM package for reporting._  |
+| `-R`/`--reporter`       | The reporter to use. In addition to all of [the `mocha` reporters](https://mochajs.org/#reporters), we also offer a [Jest-style reporter](https://jestjs.io/). _Note: we use the [`tap-mocha-reporter`](https://github.com/tapjs/tap-mocha-reporter) NPM package for Mocha reporters._  |
```

Comment:
Not strictly related, but i realized we forgot to update these docs when we implemented the Jest reporter 🙂